### DISCUSSION TITLE
Locked status

### DISCUSSION THREAD

**Discussion by skjeldgaard** [pypolestar/polestar_api#114] [Ideas]
Would be nice to be able to see if car is locked, in general or individual doors and frunk/trunk. Maybe even the ability lock/unlock doors and frunk/trunk.

**Comment by leeyuentuen** [comment]
on the mobile app, we use different auth data
polxplore but on this api we use polmystar

if we reverse engineer the mobile app, then we could do more. but the code (android) is not easy to reverse engineered ( code scrabbled)
